src/filebrowserformaction.cpp

```diff
@@ -318,7 +318,8 @@ void FileBrowserFormAction::init()
 
 	// Set position to 0 and back to ensure that the text is visible
 	draw_form();
-	set_value("filenametext_pos", std::to_string(default_filename.length()));
+	set_value("filenametext_pos",
+		std::to_string(default_filename.to_locale_string().length()));
```

Comment:
Please mark this with `TODO: #2326 use length by graphemes` and a link to this comment, because this place needs more work:

- need to check what units STFL uses for position: bytes of UTF-8, Unicode characters, graphemes?
- need to provide the length in appropriate units here

Reply:
https://github.com/newsboat/newsboat/pull/2561/commits/e8ec713bc14b28b482c997cba650e9881199ea91